content/source/docs/enterprise/private/aws-setup-guide.html.md

```diff
@@ -139,11 +139,11 @@ certificate codified during an unattended installation.
 
 If a Classic or Application Load Balancer is used, SSL/TLS will be terminated there.
 In this configuration, the PTFE instances should still be configured to listen
-for incoming SSL/TLS connections but can do so using a self-signed certificate.
+for incoming SSL/TLS connections.
 
 Since the load balancer is configured to send traffic to port 443 on the instances
-as type `https`, will ignore the fact that it can't trust the self-signed certificate
-and operate correctly.
+as type `https`, it will ignore the fact that it can't trust self-signed certificates
```

Comment:
I wonder if this bit should either be removed or say "If a self-signed certificate is used on the instance itself" as an introduction, since it doesn't directly follow on anymore.

Reply:
good call - i am happy to just remove this will defer to @hashicorp/private-tfe  on which option they'd prefer.